core/tests/protractor_utils/ExplorationEditorTranslationTab.js

```diff
@@ -165,6 +167,97 @@ var ExplorationEditorTranslationTab = function() {
       element(by.cssContainingText('option', language)).click();
   };
 
+  var selectedLanguageElement = element(
+    by.css('.protractor-test-translation-language-selector')).element(
+    by.css('option:checked'));
+
+  var languageSelectorLabelElement = element(
+    by.css('.protractor-test-language-selector-label'));
+  var progressBarLabelElement = element(
+    by.css('.protractor-test-progress-info'));
+  var translationModeButton = element(
+    by.css('.protractor-test-translation-mode'));
+  var voiceoverModeButton = element(by.css('.protractor-test-voiceover-mode'));
+
+  var saveTranslationButton = element(
+    by.css('.protractor-test-save-translation'));
+  var editTranslation = element(
```

Comment:
Typically we append the type at the end (like button, text box, etc). Can we add that here too?

Reply:
Sure, I'll change this to editTranslationButton.

Reply:
Done!